core/include/cubos/debug_render.hpp

```diff
@@ -0,0 +1,21 @@
+#ifndef CUBOS_DEBUG_RENDER_HPP
+#define CUBOS_DEBUG_RENDER_HPP
+
+#include <cubos/io/window.hpp>
+#include <glm/glm.hpp>
+
+namespace cubos
+{
+    class debug
+    {
+    public:
+        static void init(cubos::io::Window* window);
+        static void drawCube(glm::vec3 center, glm::vec3 size, float time, glm::vec3 color = glm::vec3(1));
```

Comment:
Shouldn't it be possible to set the cube's rotation? Maybe optionally send a quaternion here.

Reply:
I don't really think this is needed, would only add more optional parameters, given that a cube with a fixed orientation should suffice for debugging purposes

Reply:
I think it could also be really useful, for, for example, showing colliders which are not aligned with the axes. That parameter could just default to the identity quaternion, that way it doesn't get more complicated than needed.

Reply:
is that really what the debug renderer should be used for though? If we want to show something like colliders there should be it's own rendering system for it I believe.